modules/market/src/main/java/com/opengamma/strata/market/curve/node/FixedIborSwapCurveNode.java

```diff
@@ -56,6 +57,12 @@
    */
   @PropertyDefinition
   private final double additionalSpread;
+  /**
+   * The label to use for the node.
+   * If absent an appropriate default label will be used.
+   */
+  @PropertyDefinition(get = "optional")
```

Comment:
Why is label optional on the node but defaulted on the metadata? Is there any reason not to use a default value on the node? Is there any reason someone will want to query a node later to find out whether a label was specified?